### DISCUSSION TITLE
How trace field offset ?

### DISCUSSION THREAD

**Discussion by Alessakrendly** [vfsfitvnm/frida-il2cpp-bridge#309] [Q&A]
How trace field offset ? 
No method offset 
,Thanks

**Comment by Alessakrendly** [comment]
I know trace method 
But how trace field offset ?

**Reply by commonuserlol** [reply]
i think you can't. Just field it's variable and frida cannot attach to it (property it's diff thing)

**Reply by Alessakrendly** [reply]
> i think you can't. Just field it's variable and frida cannot attach to it (property it's diff thing)

So only method ?

**Reply by Alessakrendly** [reply]
If can doit teace field offset that good

**Reply by commonuserlol** [reply]
> > i think you can't. Just field it's variable and frida cannot attach to it (property it's diff thing)
> 
> So only method ?

maybe yes

**Reply by Alessakrendly** [reply]
> > > i think you can't. Just field it's variable and frida cannot attach to it (property it's diff thing)
> > 
> > 
> > So only method ?
> 
> maybe yes

So besides the methods, what can be traced?

**Reply by commonuserlol** [reply]
> > > > i think you can't. Just field it's variable and frida cannot attach to it (property it's diff thing)
> > > 
> > > 
> > > So only method ?
> > 
> > 
> > maybe yes
> 
> So besides the methods, what can be traced?

I mean, maybe field can be traced, but idk how

**Reply by Alessakrendly** [reply]
Yes, hopefully someone can tell us how
